File: Validus.Console/DTO/PageTemplateDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Validus.Console.DTO
{
    public class PageTemplateDto
    {
        public int Id { get; set; }
        public int PageSectionId { get; set; }
        public string TemplatedPage { get; set; }
        public string TeamTitle { get; set; }
        public string TemplateTitle { get; set; }
        public string TemplateUrl { get; set; }
        public string TemplateAfterRenderDomFunction { get; set; }
        public bool IsVisible { get; set; }
        public bool IsReadOnly { get; set; }
        public bool IsSiteStructureTemplate { get; set; }
    }
}

File: Validus.Console/Scripts/app/Dtos/PageTemplateDto.js
var ConsoleApp = window.ConsoleApp = window.ConsoleApp || {};

ConsoleApp.PageTemplateDto = function(pageSectionId, templatedPage, teamTitle, templateTitle, templateUrl, templateAfterRenderDomFunction, isVisible, isReadOnly, isSiteStructureTemplate) {
	this.PageSectionId = pageSectionId;
	this.TemplatedPage = templatedPage;
	this.TeamTitle = teamTitle;
	this.TemplateTitle = templateTitle;
	this.TemplateUrl = templateUrl;
	this.TemplateAfterRenderDomFunction = templateAfterRenderDomFunction;
	this.IsVisible = isVisible;
	this.IsReadOnly = isReadOnly;
	this.IsSiteStructureTemplate = isSiteStructureTemplate;
};